### DISCUSSION TITLE
Max Temperature for Printing Beyond 260 C

### DISCUSSION THREAD

**Discussion by HermanGaviriaMarin** [Jyers/Marlin#921] [Q&A] [answered]
Good Evening, 
I am interested in printing with materials that require higher temperatures that exceed the current maximum temperature that Jyers firmware and marling support. I understand that this is a safety feature that prevents the melting of the PTFE Bowden tube and also prevents harmful fumes from coming out of the tube. I however have an all metal hot end, which as we all know prevents the tube from melting. My question is, is there a version of this Extensible UI that allows the use and implementation of a Thermocouple or the use of a 300 C high temperature thermistor, and if there isn't one, is there plans to create one ? and if not, is there a way to have this suggested to be made ? I use an Ender 3 v2 which is what I'm sure we all use since we're here. Anyways, thanks for your time.

**Comment by smooreace** [comment]
You can compile Jyers version of Marlin, just like stock Marlin, to accept
whatever thermistor you might be using.

**Reply by HermanGaviriaMarin** [reply]
Would this create issues regarding thermal runaway ? What I'm mostly scared off is making my printer unsafe to use. And while I have your attention, is this something you have done yourself ? While compiling what should I check to make sure that thermal runaway isn't going to happen?

**Comment by smooreace** [comment, marked as answer]
I have not made these specific changes yet. But just had a 400mm AC bed
heater delivered with a new hight temp thermistor. I also have a 60W nozzle
heater and thermistor for it. The Marlin Wiki covers all of this pretty
well.

How thermal runaway is actually managed is outside my knowledge. My GUESS
would be a variation beyond some percentage of commanded temperature over a
given period of time leads to a triggering of some shutdown function... but
only a guess. That would mean that no temperature MAX will matter... its
the swing in temp that is the potential issue. Part of this is why other
higher temp thermistor are required to accurately measure that higher
temperature. The stock thermistor only functions so high.. so pushing
beyond it WOULD theoretically lead to a run-away situation. But... the
firmware obviously need to know what that thermistor is.

**Reply by Jyers** [reply]
^^^
This is exactly what it does. The firmware has a watch temp and a watch time for the bed and the hotend. If either stay at a difference of their watch temp over a span of their watch time thermal runaway will trigger. The only issue you could run into with higher temps is the time to heat being increased perhaps causing heating failures, however nothing about increasing the max temp should degrade the effectiveness of thermal runaway.
